releasenotes/notes/oracle-shared-memoy-06e54ebb7173317b.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Add shared memory (a.k.a. system global area - SGA) for Oracle databases.
```

Comment:
if we are adding a new metric, should we also document that in the release notes?

Reply:
The documentation aspect is a good question. We talked a bit about this on a previous PR and there doesn't appear to be anything like the metadata.csv that we have in agent integrations for the core agent checks. I wonder if we should still add a metadata.csv here even if it ends up not being parsed to generate documentation pages.

Reply:
I added the metric to the release notes. I'll create  `metadata.csv` at some point. We're generally lagging with documentation and will do a better job in the future. For the moment, I'm focusing to fill the feature gaps that are relevant to prospects.

Reply:
👍